org.scala.tools.eclipse.search/src/org/scala/tools/eclipse/search/indexing/Index.scala

```diff
@@ -79,6 +81,15 @@ trait Index extends HasLogger {
     location(project).toFile.exists
   }
 
+  def indexIsClean(project: IProject): Boolean = {
```

Comment:
Wouldn't it be better to name it `isIndexClean`? Reason is that `indexIsClean` seems to imply that *the index  will be clean* after calling this method.